onnxruntime/contrib_ops/cpu/skip_layer_norm.cc

```diff
@@ -110,13 +99,11 @@ void ComputeJob(
 void ComputeJob(
     const MLFloat16* input_data,
     const MLFloat16* skip_data,
```

Comment:
why need both `skip_data` and `skip_float_ptr` in the parameters? should they be converted before calling `ComputeJob` just like gamma, beta and bias?

Reply:
`skip_float_ptr` is the destination of the `MlasConvertHalfToFloatBuffer` call, and `skip_data` is used to compute the source. I wanted to make this call outside of `ComputeJob`, but the reason I can't is because the source pointer is computed using the `task_id` which I only have access to from within `ComputeJob`

Reply:
nip, maybe to convert skip_data to fp32 before parallel calling ComputeJob at where other gamma, beta etc. are converted? This way it only need one const float* skip_float_ptr.